src/components/utils/YodaMeta.js

```diff
@@ -0,0 +1,38 @@
+class YodaMeta {
+
+  // Sign In type
+  // 1 : Facebook
+  // 2 : LinkedIn
+  static LOGIN_TYPE_FB = '1';
```

Comment:
Yes, the constant exsist at both backend and frontend repo.
But since we will not integrate these repo, I think it's okay for now.